sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginReferential.java

```diff
@@ -102,6 +102,14 @@ public List<String> findLastReleasesWithDependencies(String pluginKey) {
   }
 
   public void addOutgoingDependency(Release release, String requiredPluginReleaseKey, String requiredMinimumReleaseVersion) {
+
+    // skip dependencies on license, as it's deprecated and is provided by SQ anyway. There is no need
+    // to have the explicit dependency on the update center side.
+    // Once we removed plugins < 8.9, and bundled plugins are removed from the matrix, we can drop this if
+    if("license".equals(requiredPluginReleaseKey)){
```

Comment:
nitpick: putting the literal in a constant would allow to give it a meaningful name (such as `PLUGIN_LICENSE_NAME`) and could be reused from tests

Reply:
👍